apollo-api/src/main/java/com/spotify/apollo/StatusType.java

```diff
@@ -39,6 +39,26 @@
   StatusType withReasonPhrase(String reasonPhrase);
 
   /**
+   * Convenience method for comparing the code of two StatusType instances.
+   *
+   * @param statusType the instance to compare this to
+   * @return true if code of this equals code of statusType
+   */
+  default boolean equalCode(StatusType statusType) {
+    return statusType != null && code() == statusType.code();
```

Comment:
I would agree, I think. @rydenius, is there a reason to expect/permit users to pass in a null `statusType` to this method?